packages/website/package.json

```diff
@@ -35,32 +35,21 @@
     "@typescript-eslint/eslint-plugin": "^2.33.0",
     "@typescript-eslint/parser": "^2.34.0",
     "@vue/cli": "^4.3.1",
-    "@vue/cli-plugin-babel": "^4.3.1",
```

Comment:
This is needed for our vue configuration.

Reply:
Addedback